lib/common.bash

```diff
@@ -281,3 +281,36 @@ restart_docker_service() {
 
 	return 0
 }
+
+# This function verifies that the dockerfile version is
+# equal to the test version in order to build the image or
+# just run the test
+check_dockerfiles_images()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+
+	if [ -z "$image" ] || [ -z "$dockerfile_path" ]; then
+		die "Missing image or dockerfile path variable"
+	fi
+
+	# Verify that dockerfile version is equal to test version
+	check_image=$(docker images "$image" -q)
+	if [ -n "$check_image" ]; then
+		# Check image label
+		check_image_version=$(docker image inspect $image | grep -w DOCKERFILE_VERSION | head -1 | cut -d '"' -f4)
+		if [ -n "$check_image_version" ]; then
```

Comment:
Shouldn't it be `-z` (`if [ -z "$check_image_version" ]; then`) instead of `-n`?

IIUC if `$check_image_version` is empty then the image hasn't got `DOCKERFILE_VERSION`, thus it should be built from dockerfile.

Reply:
@wainersm that is what it does if the image is not there , then it builds the image